docs/client-libraries-cluster-level-failover.md

```diff
@@ -76,6 +83,8 @@ Parameter|Default value|Required?|Description
 `failoverDelay`|N/A|Yes|The delay before the Pulsar client switches from the primary cluster to the backup cluster.<br /><br />Automatic failover is controlled by a probe task: <br />1) The probe task first checks the health status of the primary cluster. <br /> 2) If the probe task finds the continuous failure time of the primary cluster exceeds `failoverDelayMs`, it switches the Pulsar client to the backup cluster.
 `switchBackDelay`|N/A|Yes|The delay before the Pulsar client switches from the backup cluster to the primary cluster.<br /><br />Automatic failover switchover is controlled by a probe task: <br /> 1) After the Pulsar client switches from the primary cluster to the backup cluster, the probe task continues to check the status of the primary cluster. <br /> 2) If the primary cluster functions well and continuously remains active longer than `switchBackDelay`, the Pulsar client switches back to the primary cluster.
 `checkInterval`|30s|No|Frequency of performing a probe task (in seconds).
+`primaryTlsTrustCertsFilePath` |N/A|No|Path to the trusted TLS certificate file of the primary cluster.
```

Comment:
This is not the AutoClusterFailover's configuration, they are PulsarClient's configuration. I think we do not need to add here

Reply:
fix in https://github.com/apache/pulsar-site/pull/620/commits/bf4a0bedf57a1c66148426315bfc673cf354a09a